Features/full/single listing sro updates #143897499

mluedke2 opened the pull request:
Actually, don't review for a moment. I'm going to add tests...

creativecoder commented:
Yes, our stated practice is that all "public" methods should have unit tests.

creativecoder commented on app/assets/javascripts/listings/ListingController.js.coffee:
```diff
@@ -206,8 +206,28 @@ ListingController = (
+    return '1/2 bath' if numberOfBathrooms == 0.5
+
+    fullBaths = Math.floor numberOfBathrooms
+    andAHalf = numberOfBathrooms - fullBaths == 0.5
```
@mluedke2 If you were feeling clever, you could do `andAHalf = numberOfBathrooms / .5 % 2`, but what you have is probably easier to read!

mluedke2 commented:
@creativecoder Ready for review now! I moved the "bath", "people", and "person" strings into the translation file, and added tests for all public methods

mluedke2 commented:
@creativecoder Sounds like a good idea. I just pushed that refactor.

chrisdolendo commented:
:+1: thanks for the tests!
